api/delivery-schedule.md

```diff
@@ -5,85 +5,96 @@ description: >-
+
+#### Method 1: Order webhook (Push)
 
-Use the `orderEvent` field of the [order webhook](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost) endpoint.
+- Ensure your "Orders Webhook Integration" setting has "My system supports" set to "**Multiple deliveries per order line**" (this is the default)
+- Process the `orderEvent` field in the [order webhook payload](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-webhook-connector/specs.yaml#/order-webhook%20endpoints/webhookPost)
 
-The field `lines.deliverySchedule` contains the current delivery schedule of this order line.
+#### Method 2: Polling (Pull)
 
-### `deliverySchedule` fields
+Use the [Poll orders](https://swagger-ui.accp.tradecloud1.com/?url=https://api.accp.tradecloud1.com/v2/order-search/specs.yaml#/order-search/pollOrdersRoute) endpoint to periodically check for updates.
 
-* `position`: the position in the delivery schedule. Not to be confused with the `lines.position`.
+With either method, the current delivery schedule will be in the `lines.deliverySchedule` field.
+
+### Delivery schedule fields
+
+- `position`: Position number in the delivery schedule (distinct from `lines.position`)
+- `date`: Requested delivery date (ISO 8601 format `yyyy-MM-dd`)
+- `quantity`: Requested quantity (decimal format, e.g. `1234.56`)
+- `status`: [Logistics status](#logistics-status) of this delivery line according to the buyer
+- `transportMode`: Required mode of transport for goods delivery. We recommend using [UNECE.org Recommendation 19](https://tfig.unece.org/contents/recommendation-19.htm) codes.
 
 {% hint style="warning" %}
-The `position` may be unassigned in case of a delivery line split by a supplier. The buyer ERP system must assign a position to the split delivery line and update the order line to Tradecloud.
+When a supplier splits a delivery line, the `position` may be unassigned. The buyer's ERP system must assign a position to the split delivery line and update the order line in Tradecloud.
 {% endhint %}
 
-* `date`: the requested delivery date of this delivery schedule position. Date has ISO 8601 date `yyyy-MM-dd` format. See also [Standards](../../api/standards.md).
-* `quantity`: the requested quantity of this delivery schedule position. Quantity has a decimal `1234.56` format with any number of digits.
-* `status`: The [logistics status](#logistics-status) of this delivery line according to the buyer.
-* `transportMode`: The Mode of Transport used for the delivery of goods as required by the buyer. [UNECE.org Recommendation 19](https://tfig.unece.org/contents/recommendation-19.htm) is advised for Codes for Modes of Transport.
-
 ## Single delivery
 
-The alternative is to send and receive a single delivery per order line.
+The single delivery method allows only one delivery per order line.
```

Comment:
The single delivery method allows only one scheduled delivery per order line in the communications between Tradecloud and your ERP system.